contracts/src/v0.8/automation/dev/v2_3/AutomationRegistrar2_3.sol

```diff
@@ -375,8 +381,11 @@ contract AutomationRegistrar2_3 is TypeAndVersionInterface, ConfirmedOwner, IERC
       s_triggerRegistrations[params.triggerType].approvedCount++;
       upkeepId = _approve(params, hash);
     } else {
-      uint96 newBalance = s_pendingRequests[hash].balance + params.amount;
-      s_pendingRequests[hash] = PendingRequest({admin: params.adminAddress, balance: newBalance});
+      s_pendingRequests[hash] = PendingRequest({
+        admin: params.adminAddress,
+        billingToken: params.billingToken,
+        balance: params.amount
+      });
```

Comment:
I think here we could just do `s_pendingRequests[hash] = true;`

Reply:
ditto, would like some clarification.